platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -55,6 +55,16 @@ When this happens, configurations in the substituted project could be selected t
 
 Consuming non-consumable configurations in this manner is deprecated and will result in an error in Gradle 9.0.
 
+[[extending_configurations_in_same_project]]
+==== Deprecated extending configurations in the same project
+
+In prior versions of Gradle, it was possible to extend a configuration in different project.
+
+Projects should not have knowledge of configurations in other projects.
+This practice is discouraged because it can lead to unexpected behavior when configurations are extended in a way that is not intended by the configuration's owner.
```

Comment:
```suggestion
Projects should not access the mutable state of another project. 
Since Configurations are mutable, extending configurations across project boundaries restricts the parallelism that Gradle can apply.
```

Some suggestions

Reply:
That is good detail, but perhaps too much detail for some readers.  I'm going to add it and reword this a little.